src/js/background.js

```diff
@@ -140,7 +141,13 @@ Badger.prototype = {
 
   // Methods
 
+  // load seed dataset with pre-trained action and snitch maps
   loadSeedData: function() {
+    let settings = this.storage.getBadgerStorageObject("settings_map");
+    if (!settings.getItem("isFirstRun")) {
```

Comment:
I think this breaks loading seed data on demand (such as from "resetData" messages).

Reply:
Also, you could use the more compact `badger.getSettings().getItem('...` idiom instead.

Reply:
Ah, definitely. I rushed this commit in; will update in a minute.